scripts/test/environment.js

```diff
@@ -0,0 +1,25 @@
+const requestAnimationFrameTimeout = 15;
```

Comment:
What is the significance of `15`?

Reply:
No real significance tbh, it simulates the duration of a `requestAnimationFrame`, so we can predict in tests, when `requestAnimationFrame`s have completed, e.g. with a helper `waitForRequestAnimationFrame()`.